sys/conf/files

```diff
@@ -334,6 +334,8 @@ compat/cheriabi/cheriabi_vfs.c		optional compat_cheriabi
 # dtrace specific
 cddl/contrib/opensolaris/uts/common/dtrace/dtrace.c	optional dtrace compile-with "${DTRACE_C}" \
 							warning "kernel contains CDDL licensed DTRACE"
+cddl/contrib/opensolaris/uts/common/dtrace/fasttrap.c	optional dtrace compile-with "${FASTTRAP_C}"
+cddl/contrib/opensolaris/common/unicode/u8_textprep.c   optional dtrace compile-with "${FASTTRAP_C}"
```

Comment:
I don't know whether we want to make it optional on a different flag (e.g. `DTRACE_FASTTRAP`) instead of the main `DTRACE`, since it is my understanding that it was not included up until now. @brooksdavis may have stronger opinions on this.

Reply:
Good point. I think adding dtrace_fasttrap is the best thing to do (also to match the other providers)